src/containers/UserSettings/settings.tsx

```diff
@@ -13,6 +13,7 @@ import {
     THEME_KEY,
     USE_CLUSTER_BALANCER_AS_BACKEND_KEY,
     USE_DIRECTORY_OPERATIONS,
+    USE_HIDE_DOMAIN_DATABASE_KEY,
```

Comment:
Why do you add 'use' everywhere? In other cases (`USE_CLUSTER_BALANCER_AS_BACKEND_KEY`, `USE_DIRECTORY_OPERATIONS`) it's a verb, here you already have a verb 'hide'

Reply:
now key is SHOW_DOMAIN_DATABASE